flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
godot works on aarch64 too. so why limiting?

Reply:
Currently, there is only an x86_64 build. Ideally the project would be built through flatpak-builder using the godot CLI, enabling alternative architectures, however I'm not familiar with the process. Until then, the only binaries are pre-built and available from the codeberg release.